ooiui/static/js/views/asset_management/AssetView.js

```diff
@@ -6,13 +6,20 @@ var ParentAssetView = Backbone.View.extend({
      *  - derender()
      */
     initialize: function() {
-        _.bindAll(this, 'render', 'derender');
+        "use strict";
```

Comment:
@birdage , actually - not the case.  All "use strict"; should be declared in each function.